src/infra/auth/auth.ts

```diff
@@ -183,7 +183,8 @@ const getUserQuery = _.once(
 			},
 		}) as PreparedFn<
 			{ key: string },
-			Promise<Array<PickDeferred<DbUser['Read'], (typeof userFields)[number]>>>
+			Promise<Array<PickDeferred<DbUser['Read'], (typeof userFields)[number]>>>,
+			DbUser
```

Comment:
In this module there is no naming issue with 'User' vs 'DbUser' should we better us 'User' instead of DbUser?

Reply:
It looks like the clashing `User` has been removed so the code could change all the `DbUser` references not but it doesn't make sense to be part of this PR imo

Reply:
#1682